README.md

```diff
@@ -1,51 +1,72 @@
++ [Windows 10 DDK](https://msdn.microsoft.com/en-us/library/windows/hardware/ff557573(v=vs.85).aspx) - If building WebRTC Library from source (note: this is likely not the case, and is only necessary if you're planning to modify our dependency library, WebRTC)
 
-## Installing Prebuilt Libraries for /3dtoolkit-directx
+## Installing Prebuilt Dependencies
+
+> Before running our `setup.cmd` script, please ensure powershell is set to [enable unrestricted script execution](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-5.1&viewFallbackFrom=powershell-Microsoft.PowerShell.Core).
 
 After cloning this repository, run `.\setup.cmd` before opening the Project/Solution structure.
 
 This will install and configure the following:
 
 + 32bit and 64bit Debug, Release, Exes, Dlls and PDBs from this commit [Chromium m58 release](https://chromium.googlesource.com/chromium/src/+/2b7c19d3)
-+ [This patch](.\WebRTCLibs\nvencoder.patch) will be applied to the above
++ [This patch](.\WebRTCLibs\nvencoder.patch) that adds nvencode support to webrtc, and will be applied to the above
 + 32bit and 64bit Debug and Release libraries for DirectX Toolkit
 + [WebRTC-UWP](https://github.com/webrtc-uwp/webrtc-uwp-sdk) M54 synced release for UWP-based clients (Hololens)
 
 > Note: We can't currently use the [directxtk nuget packages](https://www.nuget.org/packages?q=directxtk) because they don't provide static linking targets for release builds.
 
-Once you see `Libraries retrieved and up to date` you may proceed and open [the solution](.\3dtoolkit-directx\Toolkit3D.sln).
+Once you see `Libraries retrieved and up to date` you may proceed.
 
-## Build output
+## Building the Toolkit
 
-Build the solution from Visual Studio, and look in:
+> Note: If you are using Visual Studio 2017, ensure you have installed the v140 c++ build tools, and please __do not update our projects when prompted to do so__.
 
-+ `\3dtoolkit-directx\Samples\**\Build\**` for server binaries.
-+ `\WebRTCNativeClient\Build\**` for the native client binaries.
-+ `\WebRTCLibs\**\Exe\**` for the sample signaling, turn and stun server binaries.
+> Note: To build the unity client library, you must use `Release` and `x86` for the desired configuration
+
++ Open [the 3dtoolkit solution](./3DStreamingToolKit.sln) in Visual Studio
++ Build the solution (Build -> Build Solution) in the desired configuration (Build -> Configuration Manager -> Dropdowns at the top)
++ Done!
+
+## Build output
+
+The solution builds binaries for our pluggable components and our sample implementations. All of these are output to the `Build` directory in the root of the project directory. They'll be in subfolders for the desired build configuration, in folders named according to their project name. For instance, `Client\` contains our native client, and `SpinningCubeServer\` contains our spinning cube example.
```

Comment:
Should we mention the default signaling, turn and stun server binaries in **Libraries\WebRTC\**\**\exe**?

Reply:
probably. let me see how i can rework this section to include that!